cmd/wksctl/profile/enable/enable_test.go

```diff
@@ -0,0 +1,45 @@
+package enable
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	ignoreFileFixture = `a
+b/
+ #c
+#d
+e
+f
+../g
+h..
+i # this is i
+ j/# this is j
+`
+)
+
+func TestProcessIgnoreFile(t *testing.T) {
```

Comment:
Nit:

```suggestion
func TestParseDotIgnorefile(t *testing.T) {
```

... since this is really a unit test for that method.

Reply:
Done. Thank you!